### PR TITLE
vyos: Changes to align with config_manager & Added Integration Tests

### PR DESCRIPTION
**Pull request by NilashishC**
- Fix parameters to align with config_manager role
- Changed docs
- Added load configure integration test
- Added load replace integration test

### CODE REVIEW

File: docs/config_manager/load.md [hunk 1]
```diff
@@ -43,36 +43,41 @@ inadvertantly replace your access to the device.
...
 ## Arguments
 
-### vyos_config_replace
+### config_manager_text
 
-This value enables or disables the configuration replace feature of the
-function. In order to use `vyos_config_replace` the target device must
-support config replace function.
+This value accepts the text form of the configuration to be loaded on to the remote device. 
+The configuration file should be the native set of commands used to configure the remote device
```
Comment (samccann) [inline docs/config_manager/load.md]: add a period at the end of this sentence.

File: docs/config_manager/load.md [hunk 2]
```diff
@@ -43,36 +43,41 @@ inadvertantly replace your access to the device.
...
-This value is *required*
+This value is *mutually exclusive* with `config_manager_text`
+
 
-#### Aliases
+### config_manager_replace
+
+This value enables or disables the configuration replace feature of the
+function. In order to use `config_manager_replace` the target device must
+support config replace function.
```
Comment (samccann) [inline docs/config_manager/load.md]: support the config replace...

File: docs/config_manager/load.md [hunk 3]
```diff
@@ -43,36 +43,41 @@ inadvertantly replace your access to the device.
...
+
+The default value is `False`
```
Comment (samccann) [inline docs/config_manager/load.md]: period at the end of the sentence.

File: meta/config_manager/load.yaml [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  config_manager_text:
+    description:
+      - Provide the network device configuration as a single string.  The
+        configuration text will be loaded onto the target network deivce.
+    type: str
+
+  config_manager_file:
+    description:
+      - Provide a relative or absolute path to the configuration file to be
+        loaded onto the target network device.
```
Comment (samccann) [inline meta/config_manager/load.yaml]: I'm not familiar with argument_spec, but if it allows it - would be nice to repeat the constraint here that config_manager_text is mutually exclusive with config_manager_file.

**Review by samccann — APPROVED** [review]

**Comment by NilashishC** [comment]
recheck

**Pull request merged**